docs/sources/operators-guide/tools/mimirtool.md

```diff
@@ -131,6 +131,12 @@ The commands in this section enable you to perform the following actions:
 - Interact with individual rule groups in the Mimir ruler
 - Manipulate local rule files
 
+##### Configuration
```

Comment:
```suggestion
#### Configuration
```

Reply:
Can you add a sentence mentioning that this configuration applies to all the `mimirtool rules` commands?